### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by 199943** [dk-kim/DFWSGAR#1]
Hi, I used the code you provided to test the performance of the volleyball data set, due to equipment limitations, the batch size is set to 6, other production parameters remain unchanged, the result is only 86.5, is this normal?

**Comment by dk-kim** [comment]
Hi, I assume you train original 8 class classification on Volleyball dataset using train_volleyball.sh script. 
Although we do not try to use batch size of 6 for our experiment, I think that's not the problem. We try to use batch size of 4 and 8 for our experiment but the number of batch size is not critical to final performance. 
Our method especially original 8 class classification on Volleyball dataset is sensitive to initialization so you may try different seed or just use our pretrained weights.

**Comment by 199943** [comment]
Thank you for your reply, I will try different seeds to train the network.

**Comment by gexiaojingshelby** [comment]
> 
I've met the same problem. Could  you please provide your seed as reference?

**Comment by jinsingsangsung** [comment]
Hi, I'm Jinsung Lee, the co-author of the paper.
The seed number we used for the experiment was 3.

**Comment by gexiaojingshelby** [comment]
> 
Thank you for your reply! I'll try it again!

**Comment by mt-43** [comment]
@gexiaojingshelby I'm also having this problem, do you get the results in the paper？
